calcom/cal.com
# fix: reschedule or cancel

## What does this PR do?

This pr add an option to make the cancelation reason required / optional within event types.

- Fixes #15977 

 Loom Video: https://www.loom.com/share/4c0422dd659345899d936d23ab20c7e0?sid=6bcfa671-8bcd-48fb-b983-d70c2ab01d7f


## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?
Go to advanced tab of event types and make the cancellation reason field optional / required and change the default label or placeholder if needed for the  cancel reason field.
Then create a booking and try cancelling it , you will then be able to see the changes.
